src/test/javascript/package.json

```diff
@@ -1,8 +1,7 @@
 {
   "scripts": {
     "test": "node jasmine",
-    "integration-tests": "npm test -- integration/**/*_spec.js",
-    "integration-test": "npm test -- integration/**/$FILES"
+    "integration-tests": "npm test --random=false --fail-fast=false"
```

Comment:
I added some flags here to accommodate the upgrade of jasmine, also due to changes in jasmine.js "npm run test" will run all integration tests by default

Reply:
(our pattern of for ex: npm run test -- continuous/other_test_spec.js
will continue to work just like before)